prompts.py

```diff
@@ -1,69 +1,117 @@
 import trading_algorithm
+import os
+import readline
+from pick import pick
 
-def prompt_user():
-  '''Prompts User for input to start algorithm, returns trading_terms. 
-  '''
-  ready = True
-  print(" ***********************Trading Algorithm************************ \
-    \n ***********************By William P. Fey************************ \
-    \n\n" +\
-    "This program will list a squence of buy and sell trades on \n"+ \
-    "GDAX.com based on the follosing input.\n")
-  while ready:
+def _input_default(prompt, default):
```

Comment:
since Python's native `input()` function doesn't support default values, I had to roll my own (with a little help from Stack Overflow)